webrtc-stats.html

```diff
@@ -933,10 +936,43 @@
               </dt>
               <dd>
                 <p>
-                  Represents the number of unique datachannels closed.
+                  Represents the number of unique DataChannels that have left the "open" state
+                  during their lifetime (due to being closed by either end or the underlying
+                  transport being closed). DataChannels that transition from "connecting" to
+                  "closing" or "closed" without ever being "open" are not counted in this number.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>dataChannelsRequested</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the number of unique DataChannels returned from a successful
+                  createDataChannel() call on the PeerConnection. If the underlying data transport
+                  is not established, these may be in the "connecting" state.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>dataChannelsAccepted</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the number of unique DataChannels signaled in a "datachannel" event on
+                  the PeerConnection.
                 </p>
               </dd>
             </dl>
+            <p>
+              The total number of open data channels at any time can be calculated as
+              dataChannelsOpened - dataChannelsClosed. This number is always positive.
```

Comment:
"This number is always positive." You mean non-negative? (If no data channels are ever created, 0 - 0 = 0)